src/status_im/data_store/chats.cljs

```diff
@@ -35,8 +36,13 @@
   (data-store/get-contacts chat-id))
 
 (defn has-contact?
-  [chat-id identity]
-  (data-store/has-contact? chat-id identity))
+  [{:keys [contacts]} identity]
+  (let [identities (set (map :identity contacts))]
+    (-> [identity]
+        set
+        (set/intersection identities)
+        seq
+        boolean)))
```

Comment:
Seems quite complicated for simple membership test, won't fn signature `[contacts identity]` and body:
```clj
((into #{} (map :identity) contacts) identity)
```
Work just as well ? 
Also because it's a simple helper function used just in the `protocol.handlers` ns, I would define it there.

Reply:
@janherich good point. Set is indeed can be used as function for membership test. Will change that.